bftengine/src/bftengine/messages/PartialProofsSet.cpp

```diff
@@ -208,20 +210,24 @@ class AsynchProofCreationJob : public util::SimpleThreadPool::Job {
                          IThresholdAccumulator* acc,
                          Digest& expectedDigest,
                          SeqNum seqNumber,
-                         ViewNum viewNumber) {
+                         ViewNum viewNumber,
+                         const std::string& span_context) {
     this->me = myReplica;
     this->acc = acc;
     this->expectedDigest = expectedDigest;
     this->seqNumber = seqNumber;
     this->view = viewNumber;
     this->verifier = verifier;
+    // DD: not a part of the init list to preserve the initialization order
+    span_context_ = span_context;
```

Comment:
Can you explain a bit more - I don't get what's happening here?

Reply:
The variable is initialized in the constructor but not in the initializer list.
I did it because it is the last declared in the class.
Likely, my comment is useless and it is better to remove it.